base/primes.jl

```diff
@@ -39,7 +39,7 @@ function _primesmask(lo::Int, hi::Int)
     sieve = ones(Bool, whi - wlo)
     hi < 49 && return sieve
     small_sieve = _primesmask(isqrt(hi))
-    @inbounds for i in eachindex(small_sieve)
+    @inbounds for i = 1:length(small_sieve)  # don't use eachindex here
```

Comment:
It calls a function, `wheel_prime(i)`, on `i`. This means that you need to insist on a particular meaning for `i`, in which case it's better to be explicit about what `i` needs to be.